webapp/src/AppComponents/UploadFileDialog.tsx

```diff
@@ -1,17 +1,34 @@
 import styled from "@emotion/styled";
 import { BookOpen, FileUp } from "lucide-react";
-import { type DragEvent, useRef, useState } from "react";
+import { type DragEvent, useEffect, useRef, useState } from "react";
```

Comment:
Since I am here I did the changes for the import Dialog too.

Reply:
Makes sense.